src/index.js

```diff
@@ -1487,7 +1487,7 @@ const _startRenderLoop = () => {
   const _renderLoop = async () => {
     if (args.performance) {
       if (timestamps.frames >= TIMESTAMP_FRAMES) {
-        console.log(`${(TIMESTAMP_FRAMES/(timestamps.total/1000)).toFixed(0)} FPS | ${timestamps.idle}ms idle | ${timestamps.wait}ms wait | ${timestamps.prepare}ms prepare | ${timestamps.events}ms events | ${timestamps.media}ms media | ${timestamps.user}ms user | ${timestamps.submit}ms submit`);
+        console.log(`${(TIMESTAMP_FRAMES/(timestamps.total/1000)).toFixed(0)} FPS | ${timestamps.idle/timestamps.total}ms idle | ${timestamps.wait/timestamps.total}ms wait | ${timestamps.prepare/timestamps.total}ms prepare | ${timestamps.events/timestamps.total}ms events | ${timestamps.media/timestamps.total}ms media | ${timestamps.user/timestamps.total}ms user | ${timestamps.submit/timestamps.total}ms submit`);
```

Comment:
Was the intent to divide by `TIMESTAMP_FRAMES`? I think the above will give us percentages, not milliseconds, if we do the dimensional analysis...